<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogModeling</class>
 <widget class="QDialog" name="DialogModeling">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>建模</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>模型显示</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="ModelingSubdivision" name="widget" native="true">
         <widget class="QWidget" name="widget_2" native="true">
          <property name="geometry">
           <rect>
            <x>830</x>
            <y>0</y>
            <width>91</width>
            <height>80</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(130, 150, 170);</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioButton_body">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>剖分体</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_point">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>剖分网格</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>剖分后数据</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
           <item>
            <widget class="MyTableWidget" name="tableWidget_modelData"/>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>建模参数设置</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>反演区域X起点：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_invert_begin_x"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>反演区域X终点：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_invert_end_x"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>反演区域Y起点：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_invert_begin_y"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>反演区域Y终点：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_invert_end_y"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>隧道空腔X起点：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_TunnelCavityXBegin"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>隧道空腔X终点：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_TunnelCavityXEnd"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>隧道空腔Y起点：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_TunnelCavityYBegin"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>隧道空腔Y终点：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_TunnelCavityYEnd"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>预测区域Z起点：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_invert_begin_z"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>预测区域Z终点：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_invert_end_z"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>测量电极X起点：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_MeasuringElectrode_begin_x"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>测量电极X终点：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_MeasuringElectrode_end_x"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_15">
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>测量电极Y起点：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_MeasuringElectrode_begin_y"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_16">
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>测量电极Y终点：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_MeasuringElectrode_end_y"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="100,1,1">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_edit">
             <property name="text">
              <string>编辑</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_update">
             <property name="text">
              <string>更新</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ModelingSubdivision</class>
   <extends>QWidget</extends>
   <header>ModelingSubdivision.h</header>
   <container>1</container>
   <slots>
    <slot>modelSubBodyHidden()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>MyTableWidget</class>
   <extends>QTableWidget</extends>
   <header>MyTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_body</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>modelSubBodyDisplay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>885</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_point</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>modelSubBodyHidden()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>891</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_update</sender>
   <signal>clicked()</signal>
   <receiver>DialogModeling</receiver>
   <slot>updateData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>922</x>
     <y>816</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_edit</sender>
   <signal>clicked()</signal>
   <receiver>DialogModeling</receiver>
   <slot>editData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>841</x>
     <y>816</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>-10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_invert_begin_z</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DialogModeling</receiver>
   <slot>invertBeginZTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>775</x>
     <y>669</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>-10</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>modelSubBodyHidden()</slot>
  <slot>updateData()</slot>
  <slot>editData()</slot>
  <slot>invertBeginZTextChanged(QString)</slot>
 </slots>
</ui>
